addon/templates/components/nypr-accounts/membership-card.hbs

```diff
@@ -0,0 +1,40 @@
+{{#nypr-multi-card as |m|}}
+  {{#m.panel to=1 classNames="nypr-membership-info" title="Membership Info" as |card|}}
+    {{card.header title="Membership Info"}}
+
+    {{#if pledges.isPending}}
+      {{animated-loading-icon color="#CCC" classNames="membership-loading-animation"}}
+    {{/if}}
+
+    {{#if pledges.isFulfilled}}
+      {{nypr-accounts/membership-card/current-status-detail
+        pledges=pledges
+        siteDomain=siteDomain
+        siteName=siteName}}
+    {{/if}}
+
+    <div class="pledge-tools">
+      {{#if pledges}}
+      {{#card.button classNames="giving-history-link"}}
+        <span data-test-giving-history-button>Giving History</span>
+      {{/card.button}}
+      {{/if}}
+      <div class="pledge-help-link"><span {{action (action (mut enableModal) true)}}>Help</span></div>
+    </div>
+  {{/m.panel}}
+
+  {{#m.panel to=0 title="Giving History" as |card|}}
+    {{#card.header class="giving-history-header" as |header|}}
+      {{card.button text="Back" class="nypr-card-edit-btn pledge-payment-history-back-button"}}
```

Comment:
you may not need to include `nypr-card-edit-btn` here since the contextual component already sets it (I think). if it's getting overwritten, try using the `classNames` parameter instead of `class`